src/ert/_c_wrappers/fm/ecl/ecl_run.py

```diff
@@ -246,9 +246,9 @@ def initMPI(self):
             machine_list = [localhost] * self.num_cpu
 
         self.machine_file = f"{self.base_name}.mpi"
-        with open(self.machine_file, "w") as fileH:
+        with open(self.machine_file, "w", encoding="utf-8") as filehandle:
```

Comment:
ib #3849 you renamed to f_handle. Do you think we could keep the same name here as well?

Reply:
I think I prefer to spell it out as `filehandle` whenever it does not break 88 chars on the line. If the line will be broken by `black`, I allow myself to shorten to `f_handle`. An alternative is to avoid this temporary name altogheter by using `pathlib`.